paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodsUI.kt

```diff
@@ -1,54 +1,61 @@
-package com.stripe.android.paymentsheet
+package com.stripe.android.paymentsheet.ui
 
-import androidx.annotation.RestrictTo
 import androidx.annotation.VisibleForTesting
-import androidx.compose.foundation.Image
 import androidx.compose.foundation.isSystemInDarkTheme
 import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.widthIn
 import androidx.compose.foundation.lazy.LazyListState
 import androidx.compose.foundation.lazy.LazyRow
 import androidx.compose.foundation.lazy.itemsIndexed
 import androidx.compose.foundation.lazy.rememberLazyListState
-import androidx.compose.material.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.graphics.ColorFilter
-import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.platform.LocalInspectionMode
 import androidx.compose.ui.platform.testTag
-import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
-import com.stripe.android.paymentsheet.ui.LpmSelectorText
-import com.stripe.android.paymentsheet.ui.RowButton
-import com.stripe.android.uicore.image.StripeImage
+import com.stripe.android.paymentsheet.ui.PaymentMethodsUISpacing.verticalModeSpacing
 import com.stripe.android.uicore.image.StripeImageLoader
 import com.stripe.android.uicore.strings.resolve
-import com.stripe.android.uicore.stripeColors
 
-private object Spacing {
```

Comment:
Even though `Spacing` was a private class, it would still conflict with any other class named `Spacing` in this package. Hence the rename.